src/cli/init/templates.ts

```diff
@@ -17,7 +17,31 @@ import { InitOptions } from "../types";
 
 export const { dependencies, devDependencies, peerDependencies, scripts, version } = packageJSON;
 
-export function genPkgJSON(opts: InitOptions, pgkVerOverride?: string) {
+type peprPackageJSON = {
+  data: {
+    name: string;
+    version: string;
+    description: string;
+    keywords: string[];
+    engines: { node: string };
+    pepr: {
+      uuid: string;
+      onError: string;
+      webhookTimeout: number;
+      customLabels: { namespace: { "pepr.dev": string } };
+      alwaysIgnore: { namespaces: never[] };
```

Comment:
```suggestion
      alwaysIgnore: { namespaces: string[] };
```

Shouldn't this be a string array?

Reply:
Indeed it should